frontend/taipy-gui/src/components/Taipy/Slider.tsx

```diff
@@ -30,6 +30,7 @@ interface SliderProps extends LovProps<number | string | number[] | string[], nu
     height?: string;
     min?: number;
     max?: number;
+    step?: number;
```

Comment:
the property needs to be declared on the back end too
See https://vscode.dev/github/Avaiga/taipy/blob/develop/taipy/gui/_renderers/factory.py#L428
Something like `("step", PropertyType.number, 1)` should do